### ISSUE TITLE
Broken ubuntu docker image

### ISSUE THREAD

**Issue by nicolas-chaulet** [PDAL/PDAL#2377]
Hi,
It seems that one of the latest commits has broken the ubuntu docker image build. I get the following error:
```
[95/262] Building CXX object CMakeFiles/pdal_base.dir/io/Ilvis2MetadataReader.cpp.o
FAILED: CMakeFiles/pdal_base.dir/io/Ilvis2MetadataReader.cpp.o 
/usr/bin/g++  -DUNIX -Dpdal_base_EXPORTS -I../ -Iinclude -I../vendor -I../vendor/eigen -I../vendor/pdalboost -I/usr/include/geotiff -I/build/usr/include -I/usr/include/jsoncpp -I/usr/include/x86_64-linux-gnu -I/usr/include/libxml2 -I/usr/include/gdal -isystem /build/usr/include -O3 -DNDEBUG -fPIC   -Wno-implicit-fallthrough -Wno-int-in-bool-context -Wno-dangling-else -Wno-noexcept-type -Wall -Wextra -Wpointer-arith -Wcast-align -Wcast-qual -Wno-error=parentheses -Wno-error=cast-qual -Wredundant-decls -Wno-unused-parameter -Wno-unused-variable -Wno-long-long -Wno-unknown-pragmas -Wno-deprecated-declarations -std=gnu++11 -MD -MT CMakeFiles/pdal_base.dir/io/Ilvis2MetadataReader.cpp.o -MF CMakeFiles/pdal_base.dir/io/Ilvis2MetadataReader.cpp.o.d -o CMakeFiles/pdal_base.dir/io/Ilvis2MetadataReader.cpp.o -c ../io/Ilvis2MetadataReader.cpp
../io/Ilvis2MetadataReader.cpp: In member function 'void pdal::Ilvis2MetadataReader::parseGPolygon(xmlNodePtr, pdal::MetadataNode*)':
../io/Ilvis2MetadataReader.cpp:401:5: error: 'CPLFree' was not declared in this scope
     CPLFree(polyStr);
     ^~~~~~~
../io/Ilvis2MetadataReader.cpp:401:5: note: suggested alternative: 'xmlFree'
     CPLFree(polyStr);
     ^~~~~~~
     xmlFree
```
It compiles fine on my machine though, not sure where the problem is coming from...

**Comment by abellgithub** [comment]
I assume that the GDAL that's used in the ubuntu docker container has the necessary header in some strange place.

**Comment by hobu** [comment]
We would be happy to merge a pull request that fixed the issue.

**Comment by nicolas-chaulet** [comment]
Will look into it when I have a bit more time. I tried some today but did not manage to fix it...

**Comment by nicolas-chaulet** [comment]
I assume this is fixed by #2414 ? Not sure what the fix is but thanks!
